src/test/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/service/prison/UserServiceTest.kt

```diff
@@ -287,6 +288,47 @@ class UserServiceTest {
     }
   }
 
+  @Nested
+  inner class CreateLinkedGeneralAccountToAnExistingAdminAccount {
+    @Test
+    fun `create a DPS General user linked to an Admin account`() {
+      val createLinkedGeneralUserRequest = CreateLinkedGeneralUserRequest("TEST_USER_ADM", "TEST_USER_GEN", "BXI")
+      val generalCaseLoads = listOf(
+        PrisonCaseload("NWEB", "Nomis-web Application"),
+        PrisonCaseload("BXI", "Brixton (HMP)"),
+      )
+      val adminCaseLoads = listOf(
+        PrisonCaseload("NWEB", "Nomis-web Application"),
+        PrisonCaseload("CADM_I", "Central Administration Caseload For Hmps"),
+      )
+      val generalAccount = UserCaseload(
+        "TEST_USER_GEN",
+        false,
+        PrisonUsageType.GENERAL,
+        generalCaseLoads.get(1),
+        generalCaseLoads,
+      )
+      val adminAccount =
+        UserCaseload("TEST_USER_ADM", false, PrisonUsageType.ADMIN, adminCaseLoads.get(1), adminCaseLoads)
+
+      whenever(prisonUserApiService.linkGeneralUser(createLinkedGeneralUserRequest)).thenReturn(
+        PrisonStaffUser(
```

Comment:
This PrisonStaffUser creation code is probably re-usable across different test cases. Have a look at `UserFixture` - you could probably add it there and use if from all of your tests that need it?